main.go

```diff
@@ -13,21 +13,21 @@ import (
 )
 
 var (
-	// ErrGenerateSwagger throws when fails the marshalling of the swagger struct.
-	ErrGenerateSwagger = errors.New("fail to generate swagger")
-	// ErrValidatingSwagger throws when given swagger params are not correct.
-	ErrValidatingSwagger = errors.New("fails to validate swagger")
+	// ErrGenerateOAS throws when fails the marshalling of the swagger struct.
+	ErrGenerateOAS = errors.New("fail to generate openapi")
+	// ErrValidatingOAS throws when given openapi params are not correct.
+	ErrValidatingOAS = errors.New("fails to validate openapi")
```

Comment:
Those values are exported. Maybe I'd continue to export also the old errors with a deprecation warning. What do you think about?

Reply:
makes sense